0110/streams/index.html

```diff
@@ -202,7 +202,40 @@ <h3 style="max-width: 75rem;">The easiest way to write mission-critical real-tim
         </div>
     </div>
 
-    
+    <h3 class="customer-title">See how Kafka Streams is being used</h3>
```

Comment:
The directories that correspond to version numbers are managed under the main Kafka repo under the [docs/ directory](https://github.com/apache/kafka/tree/trunk/docs) -- it's a bit annoying that the site is split up like this, but that's currently how it's managed. In order to ensure that all future versions see these changes as well, you'd create a regular Kafka PR to make the changes and then if you want it to appear immediately you'd ask for it to also be cherry picked to 0.11.0 and that the docs be redeployed (which requires a committer to build them, copy them over here, and commit).

This patch might be a little weird because the css only lives in this repo, but the rest of the patch should end up in the main Kafka repo.